pkgs/applications/science/misc/golly/default.nix

```diff
@@ -64,11 +86,14 @@ stdenv.mkDerivation rec {
     cp -r ../{Help,Patterns,Scripts,Rules} "$out/share/golly"
   '';
 
-  meta = {
+  meta = with lib; {
```

Comment:
`with lib;` is not recommended. Previous definition was okay.

Reply:
Ok